specs/engage-digital_openapi3.yaml

```diff
@@ -6746,6 +6753,9 @@ components:
           type: boolean
         published:
           type: boolean
+        rating:
+          description: is present only if the content supports rating and raiting is filled.

```

Comment:
What 'raiting' stands for? What does it mean?

Reply:
Not sure that I understand your question. It's filed in ED on some sources, I suppose you should check internal MR for more details.

Reply:
I fixed typo.